Source/Relativity.Test.Helpers/Relativity.Test.Helpers.NUnit.Integration/Mail/MailTrapMailHelperTests.cs

```diff
@@ -14,7 +18,18 @@ public class MailTrapMailHelperTests
 		/// https://mailtrap.io/signin
 		/// Also this test assumes you have an email already in the MailTrap Inbox
 		/// </summary>
+		private const string EmailTestSubject = "Relativity Integration Test";
+		private const string EmailTestBody = "Relativity test email for integration tests";
+		private const string EmailTestDisplayName = "Relativity ODA";
+		private const int EmailPort = 2525;
+		private const string EmailDomain = "smtp.mailtrap.io";
+
+		// The following below should be changed for this test
 		private const string ApiKey = "";

```

Comment:
please delete any empty constants in the class. I don't think they are needed. keeps code clean.